.markdownlint.jsonc

```
@@ -212,6 +212,13 @@
         "searchPattern": "/(\\[[^\\]]*?\\]\\(([^\\)\\n]|\\([^\\)\\n]\\)|\\s[\"'])+?)(\\n|\\s|[,:][\\s\\n])/gm",
         "searchScope": "text",
       },
+      {
+        "name": "text-code-fence",
+        "message": "Use 'plain' tag",
+        "searchPattern": "/(^ *```)(text|none|unix)/gm",
```

```suggestion
        "searchPattern": "/(^ *`{3,})(text|none|unix)/gm",
```

A code fence can have more than 3 backticks.